precode.sh

```diff
@@ -6,38 +6,42 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir -p task/dir1 task/dir2 task/dir3 task/dir3/dir4
```

Comment:
При использовании флага `-p` промежуточную папку можно не указывать
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```